lambda_strip_docx/lambda_function.py

```diff
@@ -106,23 +47,33 @@ def lambda_handler(event, context):
                 response = s3.get_object(Bucket=bucket_name, Key=object_key)
                 file_content = response['Body'].read()
 
-                # Process the DOCX file
-                try:
-                    output_bytes = strip_docx_author_metadata_from_docx(file_content)
-                except BadZipFile:
-                    logger.error(f"File {object_key} is not a valid DOCX (zip) file.")
+                extension = object_key.split(".")[-1].lower()
+                # TODO: work out what sort of file it is from magic numbers
```

Comment:
Is this TODO to be addressed in this PR or a followup?

Reply:
https://national-archives.atlassian.net/browse/FCL-1328